sdk/cognitivelanguage/azure-ai-language-conversations/README.md

```diff
@@ -12,13 +12,14 @@ Conversational Language Understanding - aka **CLU** for short - is a cloud-based
 ## _Disclaimer_
 
 _Azure SDK Python packages support for Python 2.7 ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```

Comment:
We put the special disclaimer for Python 2.7 to give customers a heads up that they may need to update their code, but we don't need to do the same for dropping Python 3.6. You can remove this comment.